timesketch/api/v1/resources/export.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2025 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Export resources for version 1 of the Timesketch API."""
+
+import json
+import logging
+
+from flask import request
+from flask import abort
+from flask import Response
+from flask import stream_with_context
+from flask import current_app
+from flask_restful import Resource
+from flask_login import login_required
+from flask_login import current_user
+
+from timesketch.api.v1 import resources
+from timesketch.lib import utils
+from timesketch.lib import forms
+from timesketch.lib.definitions import HTTP_STATUS_CODE_NOT_FOUND
+from timesketch.lib.definitions import HTTP_STATUS_CODE_FORBIDDEN
+from timesketch.lib.definitions import HTTP_STATUS_CODE_BAD_REQUEST
+from timesketch.lib.definitions import DEFAULT_SOURCE_FIELDS
+from timesketch.lib.datastores.opensearch import OpenSearchDataStore
+from timesketch.models.sketch import Sketch
+
+
+logger = logging.getLogger("timesketch.api.export")
+
+DEFAULT_POOL_MAXSIZE = 60
+
+
+class ExportListResource(resources.ResourceMixin, Resource):
```

Comment:
Had a quick chat with Janosch, given we already have `/timesketch/api/v1/export.py` having another one with export.py might be confusing, I recommend we rename this one to `stream_export` and create an issue to review if the other one can be renamed to `export_util` or something to make it clearer who is going to call it.

Reply:
I'll rename my endpoint to exportstream